lib/tasks/state_file.rake

```diff
@@ -2,6 +2,7 @@ namespace :state_file do
   desc 'Tasks for state-file'
 
   task reminder_to_finish_state_return: :environment do
+    return unless DateTime.now <= Rails.configuration.tax_deadline
```

Comment:
[note] Noting that the original requirement is to send this message once every month after april 15th (starting april 16th) but that is being moved to https://codeforamerica.atlassian.net/browse/FYST-2006

Reply:
Yep yep originally i had included it in this ticket but I took it out

Reply:
But I guess I should add that I think the other message we should put in a totally different message class and service because we will need to send it more than once and at a different cadence then this one